src/screens/ProfileScreen/index.js

```diff
@@ -14,7 +14,7 @@ import {
   View
 } from 'react-native';
 import PropTypes from 'prop-types';
-import {SafeAreaProvider} from 'react-native-safe-area-context';
+import {SafeAreaProvider, SafeAreaView} from 'react-native-safe-area-context';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been modified to include `SafeAreaView` from 'react-native-safe-area-context'. This change is consistent with the PR's goal of improving speed and fixing glitches. However, it's important to ensure that all instances of `SafeAreaProvider` in this file are replaced with `SafeAreaView` to maintain consistency.
```diff
- import {SafeAreaProvider} from 'react-native-safe-area-context';
+ import {SafeAreaProvider, SafeAreaView} from 'react-native-safe-area-context';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->